source/MRMesh/MRPlaneObject.h

```diff
@@ -46,13 +55,16 @@ class MRMESH_CLASS PlaneObject : public FeatureObject
 
     [[nodiscard]] MRMESH_API float getSizeX( ViewportId id = {} ) const;
     [[nodiscard]] MRMESH_API float getSizeY( ViewportId id = {} ) const;
+    /// calculates normalyzed directions of X,Y axis of the plane and normal as Z 
+    [[nodiscard]] MRMESH_API Matrix3f getLocalBasis( ViewportId id = {} ) const;
```

Comment:
maybe call function `calcLocalBasis` so user will see by its name that it is not free